xml/poc-install-deployer-node.xml

```diff
@@ -0,0 +1,215 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE chapter [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="poc-install-deployer-node">
+ <title>Install the Deployer Node</title>
+ <para>
+   This chapter takes you through the steps of installing the CLM deployer node.
+   If you want more in-depth instructions, see the
+  <link xlink:href="https://www.suse.com/documentation/suse-openstack-cloud-9/book_deployment/data/book_deployment.html"><citetitle>Deployment Guide using Cloud Lifecycle Manager</citetitle></link>
```

Comment:
this link 404'd for me

Reply:
How weird, cause this link works: https://www.suse.com/documentation/suse-openstack-cloud-9/book-deployment/data/book-deployment.html

Looks like the redirects are REALLY slow and only work... occasionally? I'll report this to @fsundermeyer

Reply:
Ah I see it, it's a dash instead of an underscore now. That's a genuine 404 :) thanks Jeremy!